examples/java/src/main/java/glide/examples/StandaloneExample.java

```diff
@@ -0,0 +1,128 @@
+import glide.api.models.configuration.NodeAddress;
+import glide.api.models.exceptions.ClosingException;
+import glide.api.models.exceptions.ConnectionException;
+import glide.api.models.exceptions.TimeoutException;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+
+public class StandaloneExample {
+
+    /**
+     * Creates and returns a GlideClusterClient instance. This function initializes a
+     * GlideClusterClient with the provided list of nodes.
+     *
+     * @return <code>GlideClusterClient</code> An instance of <code>GlideClusterClient</code>
+     *     connected to the discovered nodes.
+     */
+    public static GlideClient createClient() throws ExecutionException, InterruptedException {
+        String host = "localhost";
+        Integer port = 6380;
+
+        // Check GlideClusterClientConfiguration for additional options.
+        GlideClientConfiguration config =
+                GlideClientConfiguration.builder()
+                        .address(NodeAddress.builder().host(host).port(port).build())
+                        .build();
+
+        try {
+            GlideClient client = GlideClient.createClient(config).get();
+            return client;
+        } catch (ExecutionException | InterruptedException e) {
+            Logger.log(Logger.Level.ERROR, "glide", "Failed to create client: " + e.getMessage());
+        }
+        return null;
+    }
+
+    /**
+     * Executes the main logic of the application, performing basic operations such as SET, GET, and
+     * PING using the provided GlideClient.
+     *
+     * @param client An instance of <code>GlideClient</code>.
+     */
+    public static void appLogic(GlideClient client) {
+
+        // Send SET and GET
+        try {
+            CompletableFuture<String> setResponse = client.set("foo", "bar");
+            Logger.log(Logger.Level.INFO, "app", "Set response is " + setResponse.get());
+        } catch (ExecutionException | InterruptedException e) {
+            Logger.log(Logger.Level.ERROR, "glide", "Error during SET: " + e.getMessage());
+        }
+
+        try {
+            CompletableFuture<String> getResponse = client.get("foo");
+            Logger.log(Logger.Level.INFO, "app", "Get response is " + getResponse.get());
+        } catch (ExecutionException | InterruptedException e) {
+            Logger.log(Logger.Level.ERROR, "glide", "Error during GET: " + e.getMessage());
+        }
+
+        // Send PING to the primary node
+        try {
+            CompletableFuture<String> pong = client.ping();
+            Logger.log(Logger.Level.INFO, "app", "Ping response is " + pong.get());
+        } catch (ExecutionException | InterruptedException e) {
+            Logger.log(Logger.Level.ERROR, "glide", "Error during PING: " + e.getMessage());
+        }
+    }
+
+    /** Executes the application logic with exception handling. */
+    private static void execAppLogic() {
+
+        while (true) {
```

Comment:
Why do we need such loop? If GLIDE can't reach the server, it will infinetely looping and producing errors.

Reply:
That is the example provided in Python. I was asked to follow it.